README.md

```diff
@@ -13,13 +13,19 @@ Use the Azure Containers extension to quickly create and deploy containerized ap
 
 ## Create your first container app
 
-We've introduced a new command `Deploy Project from Workspace...` to simplify the process of deploying a local workspace project to a container app!  By analyzing your project directory, we automatically detect any Dockerfiles and generate all the necessary Azure resources required for creating a containerized application (monorepo support coming soon).
+We've introduced a command `Deploy Project from Workspace...` to simplify the process of deploying a local workspace project to a new container app!  By analyzing your project directory, we automatically detect any Dockerfiles and generate all the necessary Azure resources required for creating a containerized application (monorepo support now included!).
```

Comment:
I don't know if I would work this as 'We've introduced.." since we have had the command a while maybe change it to "The command `Deploy Project from Workspace` simplifies the process of deploying..." or something similar.